pkg/tfshim/sdk-v2/cty.go

```diff
@@ -28,9 +29,9 @@ func makeResourceRawConfig(
 	config *terraform.ResourceConfig,
 	resource *schema.Resource,
 ) cty.Value {
```

Comment:
Having to commit to the new method here, because "classic" parses sets incorrectly as tuples, causing type confusion in cty.Value representation.